src/main/resources/tags/block_placer_ignored_materials.json

```diff
@@ -2,16 +2,60 @@
 	"values" : [
 		"#slimefun:sensitive_materials",
 		"#slimefun:mushrooms",
+		"#minecraft:tall_flowers",
 		"#minecraft:beds",
 		"#minecraft:doors",
-		"minecraft:sugar_cane",
 		"minecraft:bamboo",
-		"minecraft:cactus",
-		"minecraft:lily_pad",
-		"minecraft:dead_bush",
+		"#minecraft:rails",
+		"#minecraft:small_flowers",
+		"#minecraft:coral_blocks",
+		"#minecraft:coral_plants",
+		"#minecraft:small_flowers",
+		"#minecraft:buttons",
+		"#minecraft:banners",
+		"#minecraft:signs",
+		"#minecraft:standing_signs",
+		"#minecraft:wall_signs",
+		"#minecraft:wooden_pressure_plates",
+		{
+			"id" : "minecraft:nether_sprouts",
+			"required" : false
+		},
+		{
+			"id" : "minecraft:crimson_roots",
+			"required": false
+		},
+		{
+			"id" : "minecraft:twisting_vines",
+			"required" : false
+		},
 		{
-			"id" : "#minecraft:tall_flowers",
+			"id" : "minecraft:warped_roots",
 			"required" : false
-		}
+		},
+		{
+			"id" : "minecraft:weeping_vines",
+			"required" : false
+		},
+		{
+			"id" : "minecraft:polished_blackstone_pressure_plate",
+			"required" : false
+		},
```

Comment:
With this being a pressure plate, it should be moved to #slimefun:sensitive_materials.
Also can you perhaps make a pressure plates tag for all of these?